### DISCUSSION TITLE
Debugging multiple screens

### DISCUSSION THREAD

**Discussion by oddlySpecificLama** [hyprwm/Hyprland#5372] [General]
Hi together,
i've looked for a bit but couldn't find any way to create multiple nested screens in a debugging sessions. I haven't found any discussion on it, neither a way how to do it the "easy" way in the code directly. Hyprland just spawns a single display when you boot it up on an already existing Hyprland session.
Due to this i sadly cannot troubleshoot my code which i'm trying to contribute, as constantly exiting the whole session, restarting it and loading in the necessary plugins is a very tedious way of troubleshooting at best. Meanwhile, of course the debugger is missing in that case, making many bugs very hard to find.

Is there already a way that i just don't know about to get more than 1 screen for troubleshooting purposes in a nested session? Or if not, would you know how to do it or know at least where in the code i could achieve this? Meanwhile, i'm also trying to figure it out on my own, but if there's an easy way, i'd most probably be able to save a good amount of time.

Many thanks in advance!

**Comment by thejch** [comment]
`WLR_BACKENDS=wayland,wayland /path/to/Hyprland`

**Reply by vaxerski** [reply]
`hyprctl output create auto` inside the nest better

**Reply by oddlySpecificLama** [reply]
Both working like a charm! Thank you guys very much!
